docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug' `1298` Fix an issue where it was not possible to add new manual balances after editing one.
```

Comment:
Was that an issue also in v1.6.1? I did not really check. If not then we don't really need a changelog. Plus note the changelog should be :bug: and not :bug'

But no worries I will fix that in the next PR. no need to push again for this.

Reply:
Okay I just checked. No need for a changelog entry. This issue is not present in 1.6.1 :)

Reply:
Yes, this issue existed in 1.6.1.

Reply:
Can you rebase and remove the changelog entry @kelsos? If not I can do it for you.

Reply:
@isidorosp Really? It existed? I added a balance with a new account in v1.6.1 and then edited it and tried to add another manual balance and it worked.

Reply:
To be honest I didn't test it but I can't see (based on the changes in the code) why this bug _wouldn't_ also exist in 1.6.1... Downloading packaged 1.6.1 now to test.

Reply:
Correction: It exists. Please just rebase on develop, fix typo and we merge.